src/index.html

```diff
@@ -8,9 +8,66 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+ 
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <div >
+        <label >
+          <input
+            class="form-search1"
```

Comment:
Try not to use number in class names
```suggestion
            class="form-search"
```